examples/customresources/demos/multiplemodules/moduletest/module_test.go

```diff
@@ -39,18 +38,7 @@ func TestMultipleModules(t *testing.T) {
 		port = portLocal
 		test.That(t, err, test.ShouldBeNil)
 
-		serverPath := testutils.BuildTempModule(t, "web/cmd/server/")
-
-		// start the viam server with a temporary home directory so that it doesn't collide with
-		// the user's real viam home directory
-		testTempHome := t.TempDir()
-		server := pexec.NewManagedProcess(pexec.ProcessConfig{
-			Name:        serverPath,
-			Args:        []string{"-config", cfgFilename},
-			CWD:         utils.ResolveFile("./"),
-			Environment: map[string]string{"HOME": testTempHome},
```

Comment:
I assume these tests that had the temp home already were changed as part of a refactor? And the other tests that omitted the `Environment` configuration were the ones conflicting?

Reply:
yep! Specifically the test that failed was https://github.com/viamrobotics/rdk/blob/a3e8dd471b6a63afbb951a73664d7f68f9d78e0e/robot/impl/discovery_test.go#L117, that uses the `setupLocalRobot` helper